contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
adding a boolean like that and having so much changed behavior isn't my favorite thing for this, it's going to be a bit error prone for every package author to potentially have to take this into account when considering where things are located